### PR TITLE
add(epp): backup exec nscp api

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Target version

- [ ] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [ ] 21.10.x (staging)
- [ ] Cloud (staging)
- [X] Plugin Packs (staging)
- [ ] 22.04.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+### Hôte
+
+* Ajoutez un hôte à Centreon depuis la page **Configuration > Hôtes**.
+* Complétez les champs **Nom**, **Alias** et **IP Address / DNS** correspondant à votre serveur **Veritas Backup Exec**.
+* Appliquez le modèle d'hôte **App-Backupexec-Nscp-Restapi-custom**.
+* Une fois le modèle appliqué, les macros ci-dessous indiquées comme requises (**Obligatoire**) doivent être renseignées.
+
+Une fois celui-ci configuré, certaines macros doivent être renseignées:
+
+| Mandatory | Name                      | Description                                                                                  |
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md:127]:
```suggestion
| Obligatoire| Nom                      | Description                                                                                  |
```

File: pp/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+title: Veritas Backup Exec NSCP Rest API
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Pack Assets
+
+### Templates
+
+The Centreon Pack Backup Exec brings 1 host template:
```
Comment (Sims24) [inline pp/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md:12]:
```suggestion
The Centreon Pack Backup Exec brings a host template:
```

File: pp/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+### Network flow
+
+The target equipment must be reachable from the Centreon poller on the TCP/8443 port.
+
+## Setup
+
+<Tabs groupId="sync">
+<TabItem value="Online License" label="Online License">
+
+1. Install the Centreon Plugin package on every Centreon poller expected to monitor **Veritas Backup Exec NSCP API** resources:
```
Comment (Sims24) [inline pp/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md:86]:
```suggestion
1. Install the Centreon plugin package on every Centreon poller expected to monitor **Veritas Backup Exec NSCP API** resources:
```

File: pp/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+yum install centreon-plugin-Applications-Protocol-Nrpe
+```
+
+2. On the Centreon web interface, install the **Veritas Backup Exec NSCP API** Centreon Pack on the **Configuration > Plugin Packs** page.
+
+</TabItem>
+
+<TabItem value="Offline License" label="Offline License">
+
+1. Install the Centreon Plugin package on every Centreon poller expected to monitor **Veritas Backup Exec NSCP API** resources:
```
Comment (Sims24) [inline pp/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md:98]:
```suggestion
1. Install the Centreon plugin package on every Centreon poller expected to monitor **Veritas Backup Exec NSCP API** resources:
```

File: pp/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+|           | NSCPRESTAPIPROTO          | Protocol used (Default: https)                                                               |
+|           | NSCPRESTAPIUSERNAME       | NSClient API username                                                                        |
+|           | NSCPRESTAPIPASSWORD       | NSClient API password                                                                        |
+|           | NSCPRESTAPILEGACYPASSWORD | NSClient API legacy authentication password                                                  |
+|           | NSCPRESTAPIEXTRAOPTIONS   | Any extra option you may want to add to the command (eg. a --insecure)                       |
+|           | BEMCLIFILE                | Powershell module file (Default: C:/Program Files/Veritas/Backup Exec/Modules/BEMCLI/bemcli) |
+
+## How to check in the CLI that the configuration is OK and what are the main options for?
+
+Once the plugin is installed, log into your Centreon Poller CLI using the
```
Comment (Sims24) [inline pp/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md:136]:
```suggestion
Once the plugin is installed, log into your Centreon poller CLI using the
```

File: pp/sidebarsPp.js [hunk 6]
```diff
@@ -315,6 +315,10 @@ module.exports = {
           type: 'doc',
           id: 'integrations/plugin-packs/procedures/applications-mulesoft-restapi'
         },
+        {
+          type: 'doc',
+          id: 'integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi'
+        },
```
Comment (Sims24) [inline pp/sidebarsPp.js:321]: Preserve alphabetical order please

**Review by Sims24 — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+|           | NSCPRESTAPIPROTO          | Protocol used (Default: https)                                                               |
+|           | NSCPRESTAPIUSERNAME       | NSClient API username                                                                        |
+|           | NSCPRESTAPIPASSWORD       | NSClient API password                                                                        |
+|           | NSCPRESTAPILEGACYPASSWORD | NSClient API legacy authentication password                                                  |
+|           | NSCPRESTAPIEXTRAOPTIONS   | Any extra option you may want to add to the command (eg. a --insecure)                       |
+|           | BEMCLIFILE                | Powershell module file (Default: C:/Program Files/Veritas/Backup Exec/Modules/BEMCLI/bemcli) |
+
+## Comment puis-je tester le Plugin et que signifient les options des commandes ?
+
+Une fois le Plugin installé, vous pouvez tester la bonne configuration NSClient directement en ligne
```
Comment (lchrdn) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md:139]:
```suggestion
Une fois le plugin installé, vous pouvez tester la bonne configuration NSClient directement en ligne
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md [hunk 8]
```diff
@@ -0,0 +1,234 @@
...
+    --command=check_version
+```
+
+La commande devrait retourner un message de sortie similaire à :
+
+```bash
+0.5.2.41 2018-04-26
+```
+
+Vous pouvez maintenant tester le Plugin Backup Exec :
```
Comment (lchrdn) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md:162]:
```suggestion
Vous pouvez maintenant tester le plugin Backup Exec :
```

File: pp/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md [hunk 9]
```diff
@@ -0,0 +1,231 @@
...
+    --command=check_version
+```
+
+The expected command output is shown below:
+
+```bash
+0.5.2.41 2018-04-26
+```
+
+Check the Backup Exec Plugin by running the following command:
```
Comment (lchrdn) [inline pp/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md:159]:
```suggestion
Check the Backup Exec plugin by running the following command:
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md [hunk 10]
```diff
@@ -0,0 +1,234 @@
...
+
+<TabItem value="Offline License" label="Offline License">
+
+1. Installez le plugin Centreon sur tous les collecteurs Centreon devant superviser des resources **Veritas Backup Exec NSCP API** :
+
+```bash
+yum install centreon-plugin-Applications-Protocol-Nrpe
+```
+
+2. Sur le serveur Central Centreon, installez le RPM du Pack **Veritas Backup Exec NSCP API** :
```
Comment (lchrdn) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md:105]:
```suggestion
2. Sur le serveur central Centreon, installez le RPM du Pack **Veritas Backup Exec NSCP API** :
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md [hunk 11]
```diff
@@ -0,0 +1,234 @@
...
+<Tabs groupId="sync">
+<TabItem value="Online License" label="Online License">
+
+1. Installez le plugin Centreon sur tous les collecteurs Centreon devant superviser des resources **Veritas Backup Exec NSCP API** :
+
+```bash
+yum install centreon-plugin-Applications-Protocol-Nrpe
+```
+
+2. Sur l'interface Web de Centreon, installez le Pack **Veritas Backup Exec NSCP API** depuis la page **Configuration > Packs de plugins**.
```
Comment (lchrdn) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-backupexec-nscp-restapi.md:93]:
```suggestion
2. Sur l'interface web de Centreon, installez le Pack **Veritas Backup Exec NSCP API** depuis la page **Configuration > Packs de plugins**.
```

**Review by Sims24 — APPROVED** [review]

**Review by lchrdn — APPROVED** [review]

**Pull request merged**